lib/oli_web/plugs/redirect_if_is_admin.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Oli.Plugs.RedirectIfIsAdmin do
```

Comment:
The name of the module isn't very representative of what it does (we know that it redirects the admin but we don't know where). WDYT about changing it to something like `DeliveryPreview` or something like that?